src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -294,7 +306,7 @@ const PermissionIdentities: FC = () => {
                 selectedNames={selectedIdentityIds}
                 setSelectedNames={setSelectedIdentityIds}
                 processingNames={[]}
-                filteredNames={nonTlsUsers.map((identity) => identity.id)}
+                filteredNames={identities.map((identity) => identity.id)}
```

Comment:
I think this should also be `nonLegacyIdentities`

Reply:
I think this comment applies to an earlier commit because this is reflected on my end.